src/components/events/EventsSection.jsx

```diff
@@ -18,23 +19,29 @@ class EventsSection extends React.Component {
   }
 
   componentDidMount() {
-    db.collection('events').onSnapshot((querySnapshot) => {
-      const events = [];
-      querySnapshot.forEach((doc) => {
-        const { eventTitle, numJudges, eventDate } = doc.data();
+    const events=[];
+    axios.get('http://localhost:5000/events').then((response) => {
+      for (var key in response.data){
```

Comment:
let's change this to be function:
```
events = response.data.map((event) => { ... }
...
```

Reply:
I tried
`let events = []
      Object.values(response.data).map((event) => { 
          events.push(event);
       })`
BUT
`const events = Object.values(response.data);`
seems to have the same function.

Notably, both methods required me to rename the table column attribute names from the current names to the names given in the API. Not sure what the impact would be on other pages.

[old naming] > [new naming from API attributes]
eventDate > event_date
eventTitle > event_title
numJudges > num_judges

Changes were made to following files so that they actually read in the values.

* EventsSection.jsx: https://github.com/uwblueprint/dancefest-web/pull/94/commits/14df67de75375d7d3c265cac721d763d08a9eb69
* EventForm.jsx: https://github.com/uwblueprint/dancefest-web/pull/94/commits/9e06fd4efd7eb68bba17a95f1e439003c9a0e178
* EventTableRow.jsx: https://github.com/uwblueprint/dancefest-web/pull/94/commits/ef745ea367be0548740f80286f9a3d0d0a2fc90d